SoftwareName_Engine/SoftwareName_Engine.csproj

```diff
@@ -0,0 +1,107 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Wonder if we should be using the newer VS project template instead, like the ones used on the engine.